presto-main/src/main/java/com/facebook/presto/sql/planner/PlanPrinter.java

```diff
@@ -211,11 +220,18 @@ public static String textDistributedPlan(List<StageInfo> stages, Metadata metada
         // it's possible that some or all of them are missing or out of date.
         // For example, a LIMIT clause can cause a query to finish before stats
         // are collected from the leaf stages.
-        Map<PlanNodeId, Long> inputPositions = new HashMap<>();
-        Map<PlanNodeId, Long> inputBytes = new HashMap<>();
-        Map<PlanNodeId, Long> outputPositions = new HashMap<>();
-        Map<PlanNodeId, Long> outputBytes = new HashMap<>();
-        Map<PlanNodeId, Long> wallMillis = new HashMap<>();
+        Map<PlanNodeId, Long> planNodeInputPositions = new HashMap<>();
+        Map<PlanNodeId, Long> planNodeInputBytes = new HashMap<>();
+        Map<PlanNodeId, Long> planNodeOutputPositions = new HashMap<>();
+        Map<PlanNodeId, Long> planNodeOutputBytes = new HashMap<>();
+        Map<PlanNodeId, Long> planNodeWallMillis = new HashMap<>();
+
+        Map<PlanNodeId, Map<String, Long>> operatorTotalDrivers = new HashMap<>();
```

Comment:
The number of drivers is useful when diagnosing concurrency and parallelism. It might quickly show some misconfiguration (task concurrency, multiple tasks per worker) or other issues.

I might remove it though if you want.